### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [MyPaint/MyPaint](https://hosted.weblate.org/projects/mypaint/mypaint/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by odysseywestra** [comment]
This is going to be a doozy...

File: po/nl.po [hunk 1]
```diff
@@ -506,10 +506,10 @@ msgid "{original_name} copy {n}"
 msgstr ""
 
 #: ../gui/brushselectionwindow.py:325
-#, fuzzy, python-brace-format
+#, python-brace-format
 msgctxt "brush group: context menu: remove from group"
 msgid "Really remove brush “{brush_name}” from group “{group_name}”?"
-msgstr "Penseel  \"{brush_name}\" echt verwijderen?"
```
Comment (odysseywestra) [inline po/nl.po:512]: Ahh it looks like it didn't include {group_name} in the translated string.

**Comment by odysseywestra** [comment]
Okay, I fixed the formatting error for that translation string. I'll leave it to you if you want to merge it to the main or your branch. If I merge to the main, you will probably need to fetch and rebase your feature branch. If you are okay with that just set your approval for merge and proceed.

**Comment by AesaraB** [comment]
Give me some time to test if these strings cause any issues if you haven't done so yourself already and I'll merge to main.

**Review by AesaraB — APPROVED** [review]
builds fine, merging it in.

**Pull request merged**